config/mongoid.yml

```diff
@@ -35,6 +59,30 @@ test:
         read:
           mode: :nearest
 
+uat:
+  clients:
+    default:
+      database: worker_uat
+      hosts:
+        - <%= ENV['MONGOID_HOST'] || 'localhost:27017' %>
+      options:
+        read:
+          mode: :primary
+        auth_source: admin
+        max_pool_size: 500
+        user: <%= ENV['MONGOID_USER'] %>
+        password: <%= ENV['MONGOID_PASSWORD'] %>
+    api:
+      database: dnz_api_uat
```

Comment:
`dnz`

Reply:
yeah all blocks in this file mention dnz hmm